examples/tfengine/team.hcl

```diff
@@ -271,7 +265,7 @@ template "project_data" {
         }]
         iam_members = {
           "roles/cloudsql.client" = [
-            "serviceAccount:$${var.bastion_service_account}",
+            "serviceAccount:bastion@example-prod-networks.iam.gserviceaccount.com",
```

Comment:
Can you remind me, why are we removing this as a variable?

Reply:
This is actually the default name: https://github.com/terraform-google-modules/terraform-google-bastion-host/blob/master/variables.tf#L130

I can set this more explicitly later so it's clearer where this value is coming from.